### PR TITLE
Evolve .NET testing targets (add net6.0)

### PR DESCRIPTION
**Pull request by drwill-ms**
netcoreapp2.1 is out of support, and we should plan for its removal but not until we've retargeted our samples. net6.0 has released and is LTS. Let's remove the former and add the latter for our samples testing.

### CODE REVIEW

File: vsts/vsts.yaml [hunk 1]
```diff
@@ -14,15 +14,15 @@ jobs:
     timeoutInMinutes: 60
     steps:
       - task: UseDotNet@2
-        displayName: 'Use .NET Core 5.0 SDK'
+        displayName: 'Use .NET 6.0 SDK'
```
Comment (drwill-ms) [inline vsts/vsts.yaml:17]: v6 and 5 are no longer named "core".

**Review by jamdavi — APPROVED** [review]

**Pull request merged**